src/openrct2/network/Network.cpp

```diff
@@ -1799,15 +1799,13 @@ void Network::Server_Handle_AUTH(NetworkConnection& connection, NetworkPacket& p
             }
         }
 
-        if ((size_t)gConfigNetwork.maxplayers <= player_list.size()) {
+        if ((size_t) gConfigNetwork.maxplayers <= player_list.size()) {
             connection.AuthStatus = NETWORK_AUTH_FULL;
-        } else
-        if (connection.AuthStatus == NETWORK_AUTH_VERIFIED) {
+        } else if (connection.AuthStatus == NETWORK_AUTH_VERIFIED) {
             connection.AuthStatus = NETWORK_AUTH_OK;
             const std::string hash = connection.Key.PublicKeyHash();
             Server_Client_Joined(name, hash, connection);
-        } else
-        if (connection.AuthStatus != NETWORK_AUTH_REQUIREPASSWORD) {
+        } else if (connection.AuthStatus != NETWORK_AUTH_REQUIREPASSWORD) {
```

Comment:
Incorrect alignment. Braces go on their own lines.

Reply:
I didn't change anything about the braces.

Reply:
You put if on the same line as else. I count that as alignment. Either don't touch alignment at all, or fix the whole line.